sql/core/src/main/scala/org/apache/spark/sql/DataFrameReader.scala

```diff
@@ -370,7 +370,8 @@ class DataFrameReader private[sql](sparkSession: SparkSession) extends Logging {
    * from values being read should be skipped.</li>
    * <li>`ignoreTrailingWhiteSpace` (default `false`): defines whether or not trailing
    * whitespaces from values being read should be skipped.</li>
-   * <li>`nullValue` (default empty string): sets the string representation of a null value.</li>
+   * <li>`nullValue` (default empty string): sets the string representation of a null value. Since
```

Comment:
Oh thanks! Indeed there are two occurrences (one in readwriter.py / one in `streaming.py`) needs fixing.